runtime/runtime_controller.cc

```diff
@@ -219,6 +219,12 @@ bool RuntimeController::NotifyIdle(fml::TimePoint deadline) {
 
   tonic::DartState::Scope scope(root_isolate);
 
+  Dart_PerformanceMode performance_mode =
```

Comment:
It might be better if we could cut this off in the animator instead, and avoid even scheduling the delayed task during this time.

However, I'm also looking into refactoring that logic a bit right now anyway, since I think it's doing duplicate work in some cases.